roles/strimzi_kafka/tasks/install.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Install Strimzi Operator
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('url', item, split_lines=False) }}"
+  with_items:
+    - "{{ kafka_install_url }}"
+
+- name: Wait for Kafka Deployment
+  kubernetes.core.k8s_info:
+    kind: Deployment
+    wait: yes
+    wait_condition:
+      type: Available
+      status: "True"
+      reason: MinimumReplicasAvailable
+    name: "{{ item }}"
+    namespace: "{{ kafka_namespace }}"
+    wait_timeout: 360
+  with_items:
+    - strimzi-cluster-operator
+
+- name: Create Kafka Cluster
```

Comment:
could move this and next task to separate `yaml` so both are `include`d from `main.yaml` with `when: kafka_create_cluster | bool`

Reply:
Done